### ISSUE TITLE
estimatefee is less than relay fee (0.15.1)

### ISSUE THREAD

**Issue by RHavar** [bitcoin/bitcoin#12450]
Just incase it's not fixed, I'm running a testnet node with 0.15.1

`estimatefee 6` returns `0.00000999`  while the `relayFee` is actually `0.00001000` -- note that this doesn't affect `estimateSmartFee` which is returning a more sane `0.00001008`.

Anyway, I realize estimatefee is deprecated but it's still probably worth chucking in a MAX(x, relayFee) there until it's removed [assuming it hasn't since been fixed]

**Comment by MarcoFalke** [comment]
Please note that

```cpp
throw JSONRPCError(RPC_METHOD_DEPRECATED, "estimatefee was removed in v0.17.\n"
```

The estimatesmartfee issue was fixed in e542728

**Comment by MarcoFalke** [comment]
Can be closed, imo.
